tests/rpmdb.at

```diff
@@ -83,7 +81,7 @@ AT_KEYWORDS([rpmdb])
 
 # This needs to run *without* RPMDB_INIT to test behavior on read-only mount
 RPMTEST_CHECK([
-rpmdb --exportdb > rdonly.list
+rpmdb --exportdb --dbpath /data/misc > rdonly.list
```

Comment:
I can reproduce the failure with a `make check` locally here. I think this is the same case as previously - we need to initialize an empty database beforehand and only then make it read-only to actually test the behavior. Not (yet) sure how to do that here, though.

Reply:
Okay, it seems like using `/var/lib/rpm-testsuite` here actually makes sense. Except we're then asserting that the file is *not* empty whereas it's of course empty since the rpmdb is :smile: 

I think this test just happened to work previously but it was never actually testing a truly read-only path.

Reply:
Nevermind, I've somehow missed that there's a prebuilt rpmdb in `/data/misc` directly in the repo, my bad.

Reply:
Yup, both "check" and "ci" pass for me locally. It was exactly like this when trying to get this test to work in the first place :sweat:

Reply:
It's strange indeed as, contrary to yours, both were failing for me consistently. A real head-scratcher. But, let's not worry about *that* here and now :sweat_smile:

Reply:
Just to leave a trail of this head-scratcher, I ran into this earlier in 4a67189f3857fa9c7a4afae768274e3706c9f87d. So whatever it is, it's consistent :laughing: